### DISCUSSION TITLE
Daily Standup (Thursday, May 26th)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#73] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x 
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Checked PR

**What are your priorities for the next 24 hours?**:
 - Help on the creation of signup and login views

**Blockers**:
 - None

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Nothing 

**What are your priorities for the next 24 hours?**:
 - Configure Login view for aspirant and company

**Blockers**:
 - None

**Shoutouts**:
 - @CristianCazares
